website/docs/shaderobjectfileoutput.md

```diff
@@ -1,12 +1,12 @@
-shaderobjectfileoutput - This page was auto-generated. Feel free to help us improve the documentation by creating a pull request.
+Specifies the output object of compiled HLSL files.
 
 ```lua
-shaderobjectfileoutput (value)
```

Comment:
Let's keep it consistent with other docs. It should be `shaderobjectfileoutput ("path")`. See: https://premake.github.io/docs/objdir/